vpd-tool/src/vpd_tool_main.cpp

```diff
@@ -0,0 +1,88 @@
+#include "constants.hpp"
+
+#include <CLI/CLI.hpp>
+
+#include <filesystem>
+#include <iostream>
+
+int main(int argc, char** argv)
+{
+    int l_retValue = 0;
+    try
+    {
+        CLI::App l_app{"VPD Command Line Tool"};
+
+        std::string l_fruPath{};
+        std::string l_recordName{};
+        std::string l_keywordName{};
+        std::string l_filePath{};
+
+        l_app.footer(
+            "Read : DBus: "
+            "vpd-tool -r -O <Object Path> -R <Record Name> -K <Keyword Name>\n"
+            "Read : Hardware : "
+            "vpd-tool -r -H -O <EEPROM Path> -R <Record Name> -K <Keyword Name>\n");
+
+        auto l_objectOption = l_app.add_option(
+            "--object, -O", l_fruPath, "Object Path or EEPROM File Path");
```

Comment:
"Inventory D-bus object path or EEPROM file path"

Reply:
Lets discuss in the team, will update once finalised.